python/lsst/ts/m2gui/controltab/tab_realtime_net_force_moment.py

```diff
@@ -0,0 +1,143 @@
+
+    Parameters
+    ----------
+    title : `str`
+        Table's title.
+    model : `Model`
+        Model class.
+
+    Attributes
+    ----------
+    model : `Model`
+        Model class.
+    net_force_moment : `list`
+        Data of the net force and moment: [fx, fy, fz, mx, my, mz]. The unit of
+        force is Newton and the unit of moment is Newton * meter.
+    """
+
+    MIN_WIDTH = 150
+
+    def __init__(self, title: str, model: Model) -> None:
+        super().__init__(title, model)
+
+        self.net_force_moment = [0.0] * 6
+
+        # Realtime figures
+        self._figures = self._create_figures()
+
+        # Timer to update the realtime figures
+        self._timer = self.create_and_start_timer(self._callback_time_out)
+
+        self._set_widget_and_layout()
+
+    def _create_figures(self, num_realtime: int = 200) -> dict:
+        """Create the figures to show the net force and moment.
+
+        Parameters
+        ----------
+        num_realtime : `int`, optional
+            Number of the realtime data (>=0). (the default is 200)
+
+        Returns
+        -------
+        figures : `dict`
+            Figures of the net force and moment.
+        """
+
+        figures = dict()
+        axes = ["fx", "fy", "fz", "mx", "my", "mz"]
+        for axis in axes:
+            label_y = "Force (N)" if axis.startswith("f") else "Moment (N * m)"
+            figures[axis] = FigureConstant(
+                1,
+                num_realtime,
+                num_realtime,
+                "Data Point",
+                label_y,
+                axis,
+                legends=[],
+                num_lines=1,
+                is_realtime=True,
+            )
+
+        for figure in figures.values():
+            figure.axis_x.setLabelFormat("%d")
+
+        return figures
+
+    @asyncSlot()
+    async def _callback_time_out(self) -> None:
+        """Callback timeout function to update the realtime figures."""
+
+        for idx, figure in enumerate(self._figures.values()):
+            figure.append_data(self.net_force_moment[idx])
+
+        self.check_duration_and_restart_timer(self._timer)
+
+    def _set_widget_and_layout(self) -> None:
+        """Set the widget and layout."""
+
+        widget = self.widget()
+        widget.setLayout(self._create_layout())
+
+        # Resize the dock to have a similar size of layout
+        self.resize(widget.sizeHint())
+
+    def _create_layout(self) -> QGridLayout:
+        """Create the layout.
+
+        Returns
+        -------
+        layout : `PySide2.QtWidgets.QGridLayout`
+            Layout.
+        """
+
+        layout = QGridLayout()
+
+        # Only draw the data with two rows
+        axes = self._figures.keys()
+        max_column = round(len(axes) / 2)
+
+        for index, axis in enumerate(axes):
+            figure = self._figures[axis]
+            figure.setMinimumWidth(self.MIN_WIDTH)
+
+            row = 0 if (index < max_column) else 1
```

Comment:
This is much easier to write as:
(row, column) = divmod(index, max_colum)

or you can just add that to addWidget:
layout.addWidget(figure, *(divmod (index, max_column)))

This is more generic, can be used in future to build widgets with number of columns.

Reply:
Fixed in the commit: [e5a46e3](https://github.com/lsst-ts/ts_m2gui/pull/45/commits/e5a46e3e907479c4ce0e51b07878165a73b2cad5) Thanks!